packages/components/psammead-media-indicator/CHANGELOG.md

```diff
@@ -3,8 +3,9 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
-| 5.0.4 | [PR#3925](https://github.com/bbc/psammead/pull/3925) Talos - Bump Dependencies - @bbc/psammead-styles, @bbc/psammead-assets |
+| 6.0.1 | [PR#?](https://github.com/bbc/psammead/pull/?) Resolve versioning mismatch |
 | 6.0.0 | [PR#3898](https://github.com/bbc/psammead/pull/3898) Migrate to Emotion |
+| 5.0.4 | [PR#3925](https://github.com/bbc/psammead/pull/3925) Talos - Bump Dependencies - @bbc/psammead-styles, @bbc/psammead-assets |
```

Comment:
Does this have the right PR linked?

Reply:
Aye it is SImon https://github.com/bbc/psammead/pull/3925/files#diff-76112a7dcb40087e6179f84ace2ced7a29f199e854891e616166a66173a486faR6

Reply:
Eugh don't think that link takes you to the highlighted section - see the v5.0.4 bump within `packages/components/psammead-media-indicator/CHANGELOG.md`